insights/parsers/softnet_stat.py

```diff
@@ -69,9 +114,11 @@ def parse_content(self, contents):
             self.cpu_nstats.append({'packet_process': stats[0],
                             'packet_drops': stats[1],
                             'time_squeeze': stats[2],
-                            'cpu_collision': stats[7],
-                            'received_rps': stats[8],
-                            'flow_limit_count': stats[9]})
+                            'cpu_collision': stats[8],
```

Comment:
It looks like the new columns were introduced to the kernel on `Sep 22, 2020`, I think there must be some servers still using the old version without the new columns. Could you please update the code to make it compatible with both the old and new versions?

Reply:
the code now handle 9, 10, 11 or 13 columns, if key doesn't have any value you get None

Reply:
Thanks for the update, it looks good to me now.